packages/runtime/src/modules/linker.ts

```diff
@@ -90,35 +88,37 @@ const linker: Linker = async (specifier, context, options = {}) => {
 const loadActualModule = async (specifier: string, options: LinkerOptions) => {
   const log = options.log || defaultLogger;
 
-  // Lookup the path from an explicit specifier first
+  // If the specifier is a path, just import it
+  if (specifier.startsWith('/')) {
+    return import(specifier);
+  }
+
+  // Otherwise resolve the specifier to a path in the repo
   let path;
   let version;
+
   if (options.modules?.[specifier]) {
     ({ path, version } = options.modules?.[specifier]);
   }
 
   if (!path && options.repo) {
-    // Try and load a matching path from the repo
     const specifierWithVersion = version
       ? `${specifier}@${version}`
       : specifier;
-    path = await getModulePath(specifierWithVersion, options.repo);
+    path = await getModuleEntryPoint(specifierWithVersion, options.repo);
```

Comment:
This is the actual fix.

Instead of looking up the path to the module root in the repo, we look up the path to the entry point. Because I'm only interested in the CJS entry point this is actually pretty trivial.